Gems/ROS2/Code/PythonTests/SmokeTests_Periodic.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+from ly_test_tools import LAUNCHERS
+from ly_test_tools.o3de.editor_test import EditorTestSuite, EditorSingleTest
+
+@pytest.mark.SUITE_periodic # Marks the test suite as being part of a Periodic test suite
+@pytest.mark.parametrize("launcher_platform", ['windows_editor']) # This test works on Windows editor
```

Comment:
I'm a bit confused here. According to [ROS 2 project requirements](https://www.o3de.org/docs/user-guide/interactivity/robotics/project-configuration/#requirements), it doesn't work on Windows.

Reply:
You're right - that was a copy-paste error from documentation